src/kontrol/kdist/cheatcodes.md

```diff
@@ -1316,6 +1376,57 @@ If the production is matched when no prank is active, it will be ignored.
         </whitelist>
 ```
 
+- `#etchAccountIfEmpty Account` - sets an Account code to a single byte '0u8' if the account is empty to circumvent the `extcodesize` check that Solidity might perform [source](https://github.com/foundry-rs/foundry/blob/b78289a0bc9df6e35624c632396e16f27d4ccb3f/crates/cheatcodes/src/evm/mock.rs#L54).
```

Comment:
Sorry, I forgot to include it in the previous review. You can probably put the link to Foundry documentation in brackets as `(source)`, so it's more obvious that it's not part of the sentence :) 

```suggestion
- `#etchAccountIfEmpty Account` - sets an Account code to a single byte '0u8' if the account is empty to circumvent the `extcodesize` check that Solidity might perform ([source](https://github.com/foundry-rs/foundry/blob/b78289a0bc9df6e35624c632396e16f27d4ccb3f/crates/cheatcodes/src/evm/mock.rs#L54)).
```